ang/crmUi.js

```diff
@@ -642,7 +642,7 @@
             $timeout(function () {
               // ex: msg_template_id adds new item then selects it; use $timeout to ensure that
               // new item is added before selection is made
-              element.select2('val', ngModel.$viewValue);
+              element.select2('val', ngModel.$modelValue);
```

Comment:
don't know that much about the diff btw viewValue and modelValue in Angular, but would note that this code is based on the `crmUiSelect` directive, and if we are changing it here should we also consider changing Line 602 above?